pkg/services/ngalert/state/state.go

```diff
@@ -155,7 +155,7 @@ func (c StateTransition) PreviousFormatted() string {
 	return FormatStateAndReason(c.PreviousState, c.PreviousStateReason)
 }
 
-func (c StateTransition) Changed() bool {
+func (c StateTransition) HasChanged() bool {
```

Comment:
Same. Refactorings are good but I would recommend to keep PRs focused. It is much simpler to review.